W1-M3-branches-and-loops-local/Program.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+
+//ExploreIf();
+
+void ExploreIf()
+{
+  int a = 5;
+  int b = 3;
+  if (a + b > 10)
+  {
+    Console.WriteLine("The answer is greater than 10");
+  }
+  else
+  {
+    Console.WriteLine("The answer is not greater than 10");
+  }
+
+  int c = 4;
+  if ((a + b + c > 10) && (a > b))
+  {
+    Console.WriteLine("The answer is greater than 10");
+    Console.WriteLine("And the first number is greater than the second");
+  }
+  else
+  {
+    Console.WriteLine("The answer is not greater than 10");
+    Console.WriteLine("Or the first number is not greater than the second");
+  }
+
+  if ((a + b + c > 10) || (a > b))
+  {
+    Console.WriteLine("The answer is greater than 10");
+    Console.WriteLine("Or the first number is greater than the second");
+  }
+  else
+  {
+    Console.WriteLine("The answer is not greater than 10");
+    Console.WriteLine("And the first number is not greater than the second");
+  }
+}
+
+/*
+ * int counter = 0;
+do
+{
+  Console.WriteLine($"Hello World! The counter is {counter}");
+  counter++;
+} while (counter < 10);
+*/
+
+/*
+for (int index=0; index < 10; index++)
+{
+  Console.WriteLine($"Hello World! The index is {index}");
+}
+*/
+/*
+ for (int row = 1; row < 11; row++)
+{
+    for (char column = 'a'; column < 'k'; column++)
+    {
+        Console.WriteLine($"The cell is ({row}, {column})");
+    }
+}
+*/
+
+
+//Find the numbers divisible by 3 from 1 to 20
+int sumOf = 0;

```

Comment:
Doesn't this show the numbers divisible from 0 to 19. 

Changed code to 

```cs
int sumOf=0;

for (int dividend=0; dividend<20; dividend++)
{
    Console.WriteLine("Dividend: " + dividend);
 if ((dividend % 3) == 0)
  {
    sumOf=sumOf+dividend;
  }
}
Console.WriteLine(sumOf);
```
got

```
Dividend: 0
Dividend: 1
Dividend: 2
Dividend: 3
Dividend: 4
Dividend: 5
Dividend: 6
Dividend: 7
Dividend: 8
Dividend: 9
Dividend: 10
Dividend: 11
Dividend: 12
Dividend: 13
Dividend: 14
Dividend: 15
Dividend: 16
Dividend: 17
Dividend: 18
Dividend: 19
63
```